crates/pallets/starknet/src/lib.rs

```diff
@@ -485,6 +489,16 @@ pub mod pallet {
 
     /// The Starknet pallet internal functions.
     impl<T: Config> Pallet<T> {
+        /// Get current block number.
+        ///
+        /// # Returns
+        ///
+        /// The current block number.
+        #[inline(always)]
+        pub fn current_block_number() -> Option<U256> {
```

Comment:
I would rather stay with only `current_block` and do the mapping in the runtime `lib.rs`

Reply:
moved to runtime lib